perch/views.py

```diff
@@ -229,8 +229,6 @@ def reference_links(doc):
     """Get reference links"""
     if doc.get('type') == 'organisation':
         for asset_id_type, link in doc.get('reference_links', {}).items():
-            # TODO: The API expects this data structure, but we should change
-            # it because there is always only one entry in "links".
             value = {
```

Comment:
Removed this TODO because it could be the case that there are multiple keys in reference_links. For example with lobster, they could have reference links for:
- LobsterPictureID
- FlickrID